content/en/about/case-studies/bluecore/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Bluecore Leverages Istio to Migrate to Kubernetes"
+linkTitle: "Bluecore Leverages Istio to Migrate to Kubernetes"
+quote: "Istio enables an approach that allows you to start building right out of the box." 
+author:
+    name: "Shray Kumar"
+    image: "/img/authors/shray-kumar.jpg"
+companyName: "Bluecore"
+companyURL: "https://bluecore.com/"
+logo: "/logos/bluecore.png"
+skip_toc: true
+skip_byline: true
+skip_pagenav: true
+doc_type: article
+sidebar_force: sidebar_case_study
+type: case-studies
+weight: 30
+---
+
+Bluecore is a multichannel personalization platform that specializes in highly personalized emails, onsite messages, and paid media ads delivered at scale. Since Bluecore's unique offering requires processing customer recommendation data as well as sending a large quantity of personalized digital communications, they deal with ingesting, processing, and sending tons of data. it's a huge workload; during a recent Black Friday, they sent over 2 billion emails.
```

Comment:
```suggestion
Bluecore is a multichannel personalization platform that specializes in highly personalized emails, onsite messages, and paid media ads delivered at scale. Since Bluecore's unique offering requires processing customer recommendation data as well as sending a large quantity of personalized digital communications, they deal with ingesting, processing, and sending tons of data. It's a huge workload; during a recent Black Friday, they sent over 2 billion emails.
```